markdown/dev/guides/code-of-conduct/our-standards/en.md

```diff
@@ -5,16 +5,16 @@ order: 20
 
 Examples of behavior that contributes to a positive environment for our community include:
 
- - Demonstrating empathy and kindness toward other people
- - Being respectful of differing opinions, viewpoints, and experiences
- - Giving and gracefully accepting constructive feedback
- - Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
- - Focusing on what is best not just for us as individuals, but for the overall community
+-   Demonstrating empathy and kindness toward other people
```

Comment:
No, there's no build that has this code in it. But since the linter and parser we use on the site are both from the Remark ecosystem, I'm not too worried.

Still, would be good to try and get a deploy preview up

Reply:
Yeah agreed, other projects like One army's precious plastic academy use netlify to do autogenerated previews. I believe they're using a different site generator than Freesewing though but hopefully it will point you in the right direction

Example - https://github.com/ONEARMY/academy/pull/159#issuecomment-1037109540

Academy - https://community.preciousplastic.com/academy

Have added an issue https://github.com/freesewing/freesewing/issues/1909 & may be worth discussing on the call today https://github.com/freesewing/freesewing/discussions/1833

Reply:
I ran this locally and it renders fine but this spacing doesn't make any sense. If I run checkdocs locally on this file, it introduces these spaces. Then if I run it again, it complains about them. Maybe this is a bug or maybe we have something misconfigured.

Reply:
I resolved my issue by using the [recommended remark flags](https://github.com/remarkjs/remark-lint#what-is-this). I updated my local `package.json` to:
```diff
-    "checkdocs": "remark ./markdown/dev/guides/best-practices/ --quiet --frail",
-    "fixdocs": "remark ./markdown/dev/guides/best-practices/ --quiet --frail --output"
+    "checkdocs": "npx remark markdown/dev/ --use remark-preset-lint-consistent --use remark-preset-lint-recommended",
+    "fixdocs": "npx remark markdown/dev/ --use remark-preset-lint-consistent --use remark-preset-lint-recommended --output
```